### PR TITLE
Add llvm-hs to the deps to test llvm-9 linkage

### PR DESCRIPTION
**Pull request by mariari**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,10 +39,27 @@ Prerequisites
...
+
+* And run the following Nix commands:
+
+  ```bash
+  nix-channel --add https://nixos.org/channels/nixpkgs-unstable
+  nix-channel --update
+  nix-env -u
+  ```
+
+* Nix will isntal
```
Comment (Acentelles) [inline README.md:62]: Typo "install"

Comment (Acentelles) [reply README.md:62]: It'll be great to be consistent with `stack` and list the same OS here too, not only arch linux

Comment (mariari) [reply README.md:62]: Agreed, however I'm unsure of the exact process on making it work on other OS's

Comment (andy-morris) [reply README.md]: I made a PR to merge into this one for the debian setup: #1081

**Review by Acentelles — APPROVED** [review]
I'll add the macOS installation bit

**Comment by Acentelles** [comment]
Do we want to revert the Nix-related changes of this PR in favor of https://github.com/anoma/juvix/pull/1098?

**Comment by mariari** [comment]
Since the history is messy we will squash this PR

**Pull request merged**